source/Junk.js

```diff
@@ -12,6 +12,7 @@ export default class Junk extends Pixi.Sprite {
         this.position.x = x
         this.position.y = y
         this.spawnTime = Date.now()
+        this.hasWeapon = Utility.randomNumber(Reference.WEAPON_RANGE.lower, Reference.WEAPON_RANGE.upper)
```

Comment:
Eventually, we'll randomize this not during the constructor, but during pickup, so we can ensure we don't give a powerup to a player who already has that powerup. :)